lib/prototypes.h

```diff
@@ -392,6 +392,7 @@ extern /*@observer@*/const char *crypt_make_salt (/*@null@*//*@observer@*/const
 /* selinux.c */
 #ifdef WITH_SELINUX
 extern int set_selinux_file_context (const char *dst_name, mode_t mode);
+extern void reset_selinux_handle ();
```

Comment:
Maybe explicit declare the number of arguments to be zero:
```c
extern void reset_selinux_handle (void);
```

Reply:
Done